daniellamarr/selfless-service-backend
# #165165418 Setup postgresql database

#### What does this PR do?
- This PR sets up a database configuration using Sequelize
#### Description of Task to be completed?
- Setup Sequelize configuration for the Postgres DB
- Add database credentials to the environment variables
#### How should this be manually tested?
- Pull this branch `ch-setup-postgresql-database-sequelize-165165418`
- Create a `.env` file in your local machine and add values to the following keys. Values should sync with the database credentials on your local machine.
```
DB_USER="database-user"
DB_PASSWORD="database-password"
DEV_DB_NAME="development-database-name"
TEST_DB_NAME="test-database-name"
```
#### What are the relevant pivotal tracker stories?
[165165418](https://www.pivotaltracker.com/story/show/165165418)
#### Screenshots (if appropriate)
N/A
